packages/animation-curve-block/src/components/BlankSlate.tsx

```diff
@@ -0,0 +1,102 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useState } from 'react';
+
+import { IconPlus24, generateRandomId } from '@frontify/fondue';
+
+import { AnimationCurve, AnimationCurveType, BlankSlateProps, defaultAnimationCurveTypeValues } from '../types';
+import { AnimationCurveFlyout, CardText } from './';
+import { BlockInjectButton } from '@frontify/guideline-blocks-shared';
+
+export const BlankSlate = ({
+    appBridge,
+    content,
+    designTokens,
+    hasBorder,
+    lineColor,
+    endpointColor,
+    gridColor,
+    canvasHeight,
+    setLocalItems,
+    setBlockSettings,
+}: BlankSlateProps) => {
+    const height = canvasHeight !== 0 ? `${canvasHeight}px` : '82px';
```

Comment:
where do this `82px` come from? could you put it as a const with a quick description?

Reply:
added a constant `BLANK_SLATE_INITIAL_HEIGHT`